platform/android/java_glue.cpp

```diff
@@ -880,7 +880,7 @@ JNIEXPORT void JNICALL Java_org_godotengine_godot_GodotLib_setup(JNIEnv *env, jo
 
 	const char **cmdline = NULL;
 	int cmdlen = 0;
-	bool use_apk_expansion = false;
+	//bool use_apk_expansion = false;
```

Comment:
You can remove it, it's parsed and used in `platform/android/java/src/org/godotengine/godot/Godot.java`.